e2e/src/restart.spec.ts

```diff
@@ -1,50 +1,47 @@
-import { describe, expect, it } from "@jest/globals";
+import {describe, expect, it} from "@jest/globals";
 import {
   getEvents,
   execDockerCommand,
-  sendXTransactions,
   waitForEvents,
   getMessageSentEventFromLogs,
   sendMessage,
+  sendTransactionsToGenerateTrafficWithInterval,
 } from "./utils/utils";
-import { getAndIncreaseFeeData } from "./utils/helpers";
-import { Wallet, ethers } from "ethers";
-// import { MessageEvent } from "./utils/types";
+import {getAndIncreaseFeeData} from "./utils/helpers";
+import {Wallet, ethers} from "ethers";
 
 const coordinatorRestartTestSuite = (title: string) => {
   describe(title, () => {
     it("When the coordinator restarts it should resume blob submission and finalization", async () => {
       const l2Account0 = new Wallet(L2_ACCOUNT_0_PRIVATE_KEY, l2Provider);
 
+      console.log("Moving the L2 chain forward to trigger conflation...");
+      const intervalId = await sendTransactionsToGenerateTrafficWithInterval(l2Account0)
```

Comment:
I think it makes more sense to stick to usage of `l2AccountForLiveness` for this